main/src/io/github/iltotore/iron/macros/package.scala

```diff
@@ -7,174 +7,171 @@ import scala.quoted.*
  * Internal macros.
  * @see [[compileTime]] for public compile-time utilities
  */
-package object macros:
```

Comment:
Since it's not a package object anymore, maybe `package.scala` should be renamed? I don't have a better name thought.

Reply:
I'm in the same boat haha, Scala 3 decided to deprecate package objects, but don't tell us how to rename them now! 😄 I think it's okay to have it named `package.scala` until we get a better name idea. What matters is that a rename won't affect users as long as the package name remains the same.